docs/section23/Rule23-9.md

```diff
@@ -40,12 +41,26 @@
+        - look at each terminal in the proposed zone: `for terminal_p in zone_p.terminals:`
+          - get the HVAC system associated with the terminal: `hvac_p = terminal_p.served_by_heating_ventilating_air_conditioning_system`
+          - get the fan system: `fan_system_p = hvac_p.fan_system`
+          - get the operation schedule hourly values: `operation_schedule_hourly_values_p = fan_system_p.operating_schedule.hourly_values`
+          - get the terminal minimum volume flowrate: `min_volume_p = terminal_p.minimum_airflow`
+          - multiply each value in the hourly schedule by the minimum airflow rate, and append this to the list of minimum volume flowrates: `minimum_volume_list_p.append([i * min_volume_p for i in operation_schedule_hourly_values_p]`
+        - sum all of the values for each hour, so that the resulting list is a list of the sum of minimum flowrates across all terminals each hour of the year: `min_volume_flow_tot_p = [sum(values) for values in zip(*minimum_volume_list_p)]`
+        - find the minimum non-zero value in the list: `min_volume_p = min(filter(lambda x: x != 0, min_volume_flow_tot_p))`
  
   **Rule Assertion:**  
   
-    - Case 1: the minimum volume flow rate is equal to the maximum of the min_ventilation_flowrate and 50% of the maximum_supply_flowrate - return PASS with a note: `if min_volume_flowrate == max(min_ventilation_flowrate, 0.5 * maximum_supply_flowrate): PASS; note = "The minimum volume flowrate is equal to the maximum of the minimum ventilation flowrate and 50% of the maximum supply flow rate.  If any airflow required to comply with codes or accredidation standards is MORE than this value, the minimum volume airflow should be set to this value.  We are not able to determine the airflow required to comply with codes or accreditation standards at this time, please double check that there are no additional codes or accreditation standards in regards to airflow." `  
-    
-    - Case 2: the minimum volume flow rate is greater than the maximum of the min_ventilation_flowrate and 50% of the maximum_supply_flowrate - return UNDETERMINED with a note: `elif min_volume_flowrate > max(min_ventilation_flowrate, 0.5 * maximum_supply_flowrate): UNDETERMINED; note = "The minimum volume flowrate is greater than the maximum of the minimum ventilation flowrate and 50% of the maximum supply flow rate.  This is correct IF the minimum volume flowrate is equal to any airflow required to comply with codes or accredidation standards, the system passes, otherwise it fails.  We are not able to determine the airflow required to comply with codes or accreditation standards at this time." `  
+    - Case 1: the minimum volume flow rate is equal to the maximum of the min_ventilation_flowrate, 50% of the maximum_supply_flowrate and the minimum supply air flowrate in the proposed model: PASS: `if min_volume_flowrate == max(min_ventilation_flowrate, 0.5 * maximum_supply_flowrate, min_volume_p): PASS`
```

Comment:
@KarenWGard I think this needs to be updated to: if minimum_value_flow rate = 0.5 * maximum_supply_flow_rate = minimum_volume_p then pass

Where did all of our comments go? I was hoping to read through them to remind me of where we left off on this.

Reply:
I think we discussed it on a call.  I got into it & remembered what we talked about.  I think it's good to go...